hosts

```diff
@@ -10000,8 +9994,7 @@
 2607:f8b0:400f:7::b r5---sn-qxo7snel.googlevideo.com
 2607:f8b0:400f:7::c r6---sn-qxo7snel.googlevideo.com
 2607:f8b0:400f:c::3 r1---sn-qxo7sner.googlevideo.com
-2607:f8b0:400f:c::4 r2---sn-qxo7sner.googlevideo.com
-2607:f8b0:4012::6 r1---sn-9gv7en7e.googlevideo.com
+2607:f8b0:400f:c::4 r2---sn-qxo7sner.googlevideo.com2607:f8b0:4012::6 r1---sn-9gv7en7e.googlevideo.com

```

Comment:
Yeah, they are concatenated together, aa344ae only fixes the file googlevideo.txt, you need to run merge_snippets.sh again to synchronize the changes to main hosts file. Otherwise, this PR will affect main file.